tests/fixtures/abridged_localpart_invalid.txt

```diff
@@ -24,11 +24,9 @@ ote"
 @
 a"bcd,e:f;g<h>i[j\k]l
 a"bcde:fg<h>i[j\k]l
-just"not"right
 this is"not\allowed
 this\ still\"not\\allowed
 partially."quoted"
-just"not"right
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `just "not" right@example.com` where `just "not"` is the display name.